scripts/cibuild/ConfigureVstsBuild.ps1

```diff
@@ -122,6 +124,12 @@ if (-not (Test-Path $regKeyNuGet) -or ($has32bitNode -and -not (Test-Path $regKe
     }
 }
 
+if (-not (Test-Path $regKeyFileSystem)) 
```

Comment:
should the build clean up at the end? 
This property will stay set on that machine.